rackcity/api/serializers/it_instance_serializers.py

```diff
@@ -1,8 +1,35 @@
 from rest_framework import serializers
 from rackcity.models import ITInstance
+from .it_model_serializers import ITModelSerializer
+from .rack_serializers import RackSerializer
 
 
 class ITInstanceSerializer(serializers.ModelSerializer):
+    """
+    Serializes all fields on ITInstance model, where model and rack fields are
+    defined by their pk only.
```

Comment:
want this for /add. don't want to have to pass in entirely defined model when creating a new instance, should only have to specify pk of model